metadetect/lsst_metadetect.py

```diff
@@ -217,9 +230,10 @@ def add_ormask(ormask, res):
     copy in ormask values using the row, col positions
     """
     for i in range(res.size):
-        res['ormask'][i] = ormask[
-            int(res['row'][i]), int(res['col'][i]),
-        ]
+        local_row = int(res['row'][i] - res['row0'][i])
+        local_col = int(res['col'][i] - res['col0'][i])
```

Comment:
The nearest pixel is best gotten by `int(np.floor(x+0.5))`. Otherwise, you are asymmetrically pushing things to the left by half a pixel.